components/calendar/tests/monotonic_year.rs

```diff
@@ -0,0 +1,72 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use icu_calendar::AnyCalendar;
+use icu_calendar::AnyCalendarKind;
+use icu_calendar::Date;
+
+#[cfg(test)]
+use std::rc::Rc;
+
+/// Reference: <https://tc39.es/proposal-intl-era-monthcode/#sec-temporal-calendardatearithmeticyear>
+static MONOTONIC_EPOCHS: &[(AnyCalendarKind, i32)] = &[
```

Comment:
I worry about hardcoding these values. Aren't they the same as in the `provider/source/src/calendar/eras.rs` test, but without day accuracy?

Reply:
The goal is to match the Temporal spec.  That's distinct from `eras.rs`.